auth-external-api/src/main/java/uk/gov/di/authentication/external/lambda/UserInfoHandler.java

```diff
@@ -130,6 +131,17 @@ public APIGatewayProxyResponseEvent userInfoRequestHandler(APIGatewayProxyReques
             throw new RuntimeException(e);
         }
 
+        AuthSessionItem authSession;
+
+        Optional<AuthSessionItem> optionalAuthSession =
+                authSessionService.getSessionFromRequestHeaders(input.getHeaders());
+
+        if (optionalAuthSession.isEmpty()) {
+            LOG.warn("Session cannot be found");
+            return generateApiGatewayProxyErrorResponse(400, ErrorResponse.ERROR_1000);
```

Comment:
Same comment as previous. Also any reason we are logging here but not in the other handlers?

Reply:
Yeah not sure why this logging is here, will remove because I realise it could be a bit misleading if the sessionId header is not present